keyboards/linworks/fave104/keymaps/default/keymap.c

```diff
@@ -0,0 +1,19 @@
+#include QMK_KEYBOARD_H
```

Comment:
Could you add a GPL2+ compatible license header here?  All of the source files need to have a license explicitly listed.

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

Reply:
It was deleted by my mistake. I'll try to fix it.